detect/region.go

```diff
@@ -0,0 +1,62 @@
+package detect
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/urfave/cli"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/utils"
+
```

Comment:
Are you sure your IDE is correctly configured?
```suggestion
```

Reply:
I will fix that and have a look at my linter configuration.